src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -430,4 +431,29 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
   <data name="AddPkg_UserSpecified" xml:space="preserve">
     <value>user specified</value>
   </data>
+  <data name="Info_RemovePkgRemovingReference" xml:space="preserve">
+    <value>Removing PackageReference for package '{0}' from project '{1}'</value>
+    <comment>{0} - Package Id
+{1} - Project Path</comment>
+  </data>
+  <data name="RemovePkg_Description" xml:space="preserve">
+    <value>Removes a package reference from a project.</value>
+  </data>
+  <data name="RemovePkg_PackageIdDescription" xml:space="preserve">
+    <value>Id of the package to be removed.</value>
+  </data>
+  <data name="RemovePkg_ProjectPathDescription" xml:space="preserve">
+    <value>Path to the project.</value>
+  </data>
+  <data name="Error_UpdatePkgNoSuchPackage" xml:space="preserve">
+    <value>Project '{0}' does not contain any PackageReference '{1}' to '{2}'</value>
```

Comment:
These strings might be more clear if the operation wasn't in quotes. Since this is an internal string it should be safe, it isn't like free form user input where it might change the whole meaning of the sentence otherwise.

Do you think localization will have issues trying to fit these parameters in?